### PR TITLE
Remove CH and use simple header

### PR DESCRIPTION
**Pull request by alex-deaconu**
Removed CH and dependencies
Add segmentAnalytics on their own

Remove Google Analytics code

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: _includes/header-navbar.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<nav class="navbar navbar-default" ng-controller="HeaderNavbarCtrl">
+  <div class="container">
+    <div class="navbar-header" style="width: 100%">
+      <div class="pull-right">
+        <a href="https://apps.risevision.com/signin" target="_blank" class="btn btn-lg btn-primary">Sign Up Free <i class="fa fa-google"></i></a>
+        <a href="https://apps.risevision.com/signup" target="_blank" class="btn btn-lg btn-link">Sign In</a>
```
Comment (ezequielc) [inline _includes/header-navbar.html]: @alex-deaconu Are Sign Up and Sign In links swapped? Cheers

Comment (alex-deaconu) [inline _includes/header-navbar.html]: @ezequielc oops. Should be fixed. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu :+1:

**Pull request merged**